src/index.html

```diff
@@ -1,14 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <main class="container">
+      <div class="block" id="qa-block">
+        <div class="blue">
+          <div class="rubbish">
+            <div class="yellow">
+              <div class="rubbish">
+                <div class="text">
+                  In programming languages, a closure is a technique for implementing lexically
+                  scoped name binding in a language with first-class functions. Operationally, a
+                  closure is a record storing a function together with an environment.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="blue">
+          <div class="rubbish">
+            <div class="yellow">
+              <div class="rubbish">
+                <div class="text">
+                  У програмуванні замиканням називають підпрограму, що виконується в середовищі,
+                  яке містить одну або більше зв’язаних змінних. Підпрограма має доступ до цих
+                  змінних під час виконання.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="blue">
+          <div class="rubbish">
+            <div class="yellow">
+              <div class="rubbish">
+                <div class="text">
```

Comment:
Html елемент може мати кілька
```suggestion
      <div class="block blue rubbish">
            <div class="yellow rubbish">
                <div class="text rubbish">
```